.circleci/config.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           name: Show diff
           command: |
             git --no-pager diff
+            git diff --quiet
```

Comment:
For context, why are both of these lines necessary?

Reply:
I added `git diff --quiet` to make this step fail when there are differences, but I just notice we can use `--exit-code
`.

```suggestion
            git --no-pager diff --exit-code
```

Reply:
Fixed!